### PR TITLE
Clarify SWIFT_USE_SCCACHE meaning

### PR DESCRIPTION
**Pull request by dabrahams**
It wasn't clear from the text what "will pick it up" means.

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci smoke test Linux

**Comment by CodaFi** [comment]
@swift-ci smoke test Linux

File: docs/DevelopmentTips.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ $ sccache --start-server
 $ ./swift/utils/build-script MY_ARGS --sccache
 ```
 
-If you want to always use sccache, you can `export SWIFT_USE_SCCACHE=1` and the build script will pick it up.
+If you want to always use sccache, you can `export SWIFT_USE_SCCACHE=1` and omit the `--sccache` flag from the `build-script` invocation (you'll still need to start the server).
```
Comment (dabrahams) [inline docs/DevelopmentTips.md:39]:
```suggestion
If you want to always use sccache, you can `export SWIFT_USE_SCCACHE=1` and omit the `--sccache` flag from the `build-script` invocation.
```

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
@swift-ci smoke test Linux

**Pull request merged**